benchconnect/benchconnect/utils.py

```diff
@@ -0,0 +1,48 @@
+from json import dumps, load, loads
+from pathlib import Path
+from typing import Any, Dict, List
+
+import click
+
+ENV_VAR_HELP = """
+
+\b
+Environment variables:
+  CONBENCH_URL: The URL of the Conbench server. Required.
+  CONBENCH_EMAIL: The email to use for Conbench login. Only required if the server is private.
+  CONBENCH_PASSWORD: The password to use for Conbench login. Only required if the server is private.
+"""
+
+
+def load_json(json: str, path: str, ndjson: str) -> List[Dict[str, Any]]:
+    "Load JSON from a string, file path, directory path, or stdin"
+
+    stdin = click.get_text_stream("stdin")
+    if not ndjson and not stdin.isatty():
+        ndjson = stdin.read()
+
```

Comment:
We have to be careful with logging and warning so it goes to stderr, because stdout needs to be only output where there is any. But I think I can make this work